concent_api/concent_api/urls.py

```diff
@@ -1,24 +1,16 @@
-"""concent_api URL Configuration
+from django.conf            import settings
+from django.core.exceptions import ImproperlyConfigured
 
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.11/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.conf.urls import url, include
-    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
-"""
-from django.conf.urls import url, include
-from django.contrib   import admin
+from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
 
-import core.urls
+urlpatterns = []
```

Comment:
It would be more readable if you move this to line 14, just before the first usage of this variable.

Reply:
done